jwql/database/database_interface.py

```diff
@@ -208,7 +208,7 @@ class Monitor(base):
     monitor_name = Column(String(), nullable=False)
     start_time = Column(DateTime, nullable=False)
     end_time = Column(DateTime, nullable=True)
-    status = Column(Enum('SUCESS', 'FAILURE'), nullable=True)
+    status = Column(Enum('SUCESS', 'FAILURE', name='monitor_status'), nullable=True)
```

Comment:
Note that this is a small bug fix that doesn't have anything to do with `reset_database.py`